src/python/grpcio/grpc/experimental/aio/_interceptor.py

```diff
@@ -388,6 +433,111 @@ def __await__(self):
         return response
 
 
+class _InterceptedStreamResponseMixin:
+    _response_aiter: AsyncIterable[ResponseType]
+
+    def _init_stream_response_mixin(self) -> None:
+        self._response_aiter = self._wait_for_interceptor_task_response_iterator(
+        )
```

Comment:
AsyncIO has a weird behavior that async generator are treated as coroutine. So, if there isn't a consumer for the `self._response_aiter`, it will log a warning saying that a coroutine is deallocated without `await`. A workaround is only instantiate the coroutine when `__aiter__` is called.